html/feed/feed.js

```diff
@@ -1,5 +1,369 @@
+import timeStamp from "../utils/helpers/timeStamp.js";
+import postComment from "../utils/helpers/postComment.js";
+import reactToPosts from "../utils/helpers/reactToPosts.js";
+import deletePost from "../utils/helpers/deletePost.js"; 
+import editPost from "../utils/helpers/editPost.js"; 
+
+const feedContainer =document.querySelector ("#feed-container")
+const confirmEditBtn = document.querySelector ("#confirmEdit")
+
+let token 
 const baseUrl = "https://api.noroff.dev/api/v1";
 
+let userName 
+token = localStorage.getItem ('bearerToken',token); 
+userName = localStorage.getItem ('name',userName); 
+
+const options = {
+  method: "GET", 
+  credentials: "same-origin", 
+  headers: {
+    "Content-Type": "application/json", 
+    Authorization: `Bearer ${token}`
+  }
+}
+
+
+async function getPosts (headerOptions){
+  try {
+      const response = await fetch (`${baseUrl}/social/posts?_author=true&_comments=true&_reactions=true`,headerOptions); 
+      const posts= await response.json();
+      console.log (posts); 
```

Comment:
Remember to remove console.logs when you want to push to Main branch as debugging should have been completed and the app doesnt get overwhelemed with logs at the end :p

Reply:
fixed